pkgs/development/coq-modules/vcfloat/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, mkCoqDerivation, coq, interval, compcert, flocq, bignums, version ? null
+}:
+
+with lib;
+mkCoqDerivation rec {
+  pname = "vcfloat";
+  owner = "VeriNum";
+  inherit version;
+  defaultVersion = with versions;
+    switch coq.coq-version [{
+      case = range "8.16" "8.17";
+      out = "2.1.1";
+    }] null;
+  release."2.1.1".sha256 =
+    "sha256-bd/XSQhyFUAnSm2bhZEZBWB6l4/Ptlm9JrWu6w9BOpw=";
+  releaseRev = v: "v${v}";
+  propagatedBuildInputs = [ interval compcert flocq bignums ];
+  meta = {
+    description =
+      "VCFloat is a tool for Coq proofs about floating-point round-off error.";
+    maintainers = with maintainers; [ quinn-dougherty ];
+  };
+}
```

Comment:
Is this auto-formatted with some kind of auto-formatter? I would prefer if you could follow a closer formatting style to what is currently done for Coq packages (see e.g., the example from the [documentation](https://nixos.org/manual/nixpkgs/unstable/#sec-language-coq)). In particular, make the `defaultVersion` more compact, add empty lines to give some space (that makes reading easier), do not add newlines between `=` and strings (release hash and description).

This is also missing a `license` field.

Reply:
thanks! I was careful to disable format on save for preexisting files, didn't know it was a priority for new files. Will cover for next commit.